src/constants/textResponses.ts

```diff
@@ -122,6 +122,17 @@ export const phrases: ResponseRepository = [
 	"The river Temarc... in winter",
 	"You know, back when I was in the academy, we would follow every toast with a song!",
 
+	// Wonka
+	"We are the music makers and we are the dreamers of the dreams.",
```

Comment:
Would you sort these quotes alphabetically?

Reply:
good advice, will do